### PR TITLE
[Docathon][Add CN Doc No.26]

### PR DESCRIPTION
**Pull request by Kaedeharai**
### PR types

Others

### PR changes

Docs

### Description

补充缺失的中文api文档 ：
#6193 

添加中文文档：
paddle.nn.functional.scaled_dot_product_attention

英文文档链接：https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/nn/functional/scaled_dot_product_attention_en.html

@sunzhongkai588 @liuzhenhai93

### CODE REVIEW

**Review by liuzhenhai93 — APPROVED** [review]
LGTM

File: docs/api/paddle/nn/functional/scaled_dot_product_attention_cn.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - **value** (Tensor) - 注意力模块中的值张量。
+                           具有以下形状的四维张量:
+                           [batch_size, seq_len, num_heads, head_dim]。
+                           数据类型可以是 float61 或 bfloat16。
+    - **attn_mask** (Tensor, 可选) - 与添加到注意力分数的 ``query``、 ``key``、 ``value`` 类型相同的浮点掩码, 默认值为空。
+    - **dropout_p** (float) - ``dropout`` 的比例, 默认值为 0.00 即不进行正则化。
+    - **is_causal** (bool) - 是否启用因果关系, 默认值为 False 即不启用。
+    - **training** (bool): - 是否处于训练阶段, 默认值为 True 即处于训练阶段。
+    - **name** (str, 可选) - 默认值为 None。通常不需要用户设置此属性。欲了解更多信息, 请参阅
+                            :ref:`api_guide_Name`。
```
Comment (sunzhongkai588) [inline docs/api/paddle/nn/functional/scaled_dot_product_attention_cn.rst]:
```suggestion
    - **name** (str, 可选) - 默认值为 None。通常不需要用户设置此属性。欲了解更多信息, 请参阅:ref:`api_guide_Name`。
```

File: docs/api/paddle/nn/functional/scaled_dot_product_attention_cn.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                           [batch_size, seq_len, num_heads, head_dim]
+                           数据类型可以是 float61 或 bfloat16。
+    - **key** (Tensor) - 注意力模块中的关键张量。
+                         具有以下形状的四维张量:
+                         [batch_size, seq_len, num_heads, head_dim]。
+                         数据类型可以是 float61 或 bfloat16。
+    - **value** (Tensor) - 注意力模块中的值张量。
+                           具有以下形状的四维张量:
+                           [batch_size, seq_len, num_heads, head_dim]。
+                           数据类型可以是 float61 或 bfloat16。
```
Comment (sunzhongkai588) [inline docs/api/paddle/nn/functional/scaled_dot_product_attention_cn.rst]:
```suggestion
    - **value** (Tensor) - 注意力模块中的值张量。具有以下形状的四维张量: [batch_size, seq_len, num_heads, head_dim]。数据类型可以是 float61 或 bfloat16。
```

File: docs/api/paddle/nn/functional/scaled_dot_product_attention_cn.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+::::::::::
+
+    - **query** (Tensor) - 注意力模块中的查询张量。
+                           具有以下形状的四维张量：
+                           [batch_size, seq_len, num_heads, head_dim]
+                           数据类型可以是 float61 或 bfloat16。
+    - **key** (Tensor) - 注意力模块中的关键张量。
+                         具有以下形状的四维张量:
+                         [batch_size, seq_len, num_heads, head_dim]。
+                         数据类型可以是 float61 或 bfloat16。
```
Comment (sunzhongkai588) [inline docs/api/paddle/nn/functional/scaled_dot_product_attention_cn.rst]:
```suggestion
    - **key** (Tensor) - 注意力模块中的关键张量。具有以下形状的四维张量:[batch_size, seq_len, num_heads, head_dim]。数据类型可以是 float61 或 bfloat16。
```

File: docs/api/paddle/nn/functional/scaled_dot_product_attention_cn.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    此 API 仅支持数据类型为 float16 和 bfloat16 的输入。
+
+
+参数
+::::::::::
+
+    - **query** (Tensor) - 注意力模块中的查询张量。
+                           具有以下形状的四维张量：
+                           [batch_size, seq_len, num_heads, head_dim]
+                           数据类型可以是 float61 或 bfloat16。
```
Comment (sunzhongkai588) [inline docs/api/paddle/nn/functional/scaled_dot_product_attention_cn.rst]:
```suggestion
    - **query** (Tensor) - 注意力模块中的查询张量。具有以下形状的四维张量：[batch_size, seq_len, num_heads, head_dim]。数据类型可以是 float61 或 bfloat16。
```

File: docs/api/paddle/nn/functional/scaled_dot_product_attention_cn.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    - **training** (bool): - 是否处于训练阶段, 默认值为 True 即处于训练阶段。
+    - **name** (str, 可选) - 默认值为 None。通常不需要用户设置此属性。欲了解更多信息, 请参阅
+                            :ref:`api_guide_Name`。
+
+
+返回
+::::::::::
+
+    - ``out`` (Tensor): 形状为 ``[batch_size, seq_len, num_heads, head_dim]`` 的 4 维张量。
+                        数据类型可以是 float16 或 bfloat16。
```
Comment (sunzhongkai588) [inline docs/api/paddle/nn/functional/scaled_dot_product_attention_cn.rst]:
```suggestion
    - ``out`` (Tensor): 形状为 ``[batch_size, seq_len, num_heads, head_dim]`` 的 4 维张量。数据类型可以是 float16 或 bfloat16。
```

**Review by sunzhongkai588 — COMMENTED** [review]
渲染有些错，详细看 [预览](http://preview-pr-6390.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/functional/scaled_dot_product_attention_cn.html)

其实英文文档也有一处错误，之后有空再改吧
image

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

File: docs/api/paddle/nn/Overview_cn.rst [hunk 6]
```diff
@@ -196,6 +196,7 @@ Transformer 相关
 
 
     " :ref:`paddle.nn.MultiHeadAttention <cn_api_paddle_nn_MultiHeadAttention>` ", "多头注意力机制"
+    " :ref:`paddle.nn.functional_scaled_dot_product_attention <cn_api_paddle_nn_functional_scaled_dot_product_attention>` ", "点乘注意力机制，并在此基础上加入了对注意力权重的缩放"
```
Comment (sunzhongkai588) [inline docs/api/paddle/nn/Overview_cn.rst]:
```suggestion
    " :ref:`paddle.nn.functional.scaled_dot_product_attention <cn_api_paddle_nn_functional_scaled_dot_product_attention>` ", "点乘注意力机制，并在此基础上加入了对注意力权重的缩放"
```

File: docs/api/paddle/nn/functional/scaled_dot_product_attention_cn.rst [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    - **is_causal** (bool) - 是否启用因果关系, 默认值为 False 即不启用。
+    - **training** (bool): - 是否处于训练阶段, 默认值为 True 即处于训练阶段。
+    - **name** (str, 可选) - 默认值为 None。通常不需要用户设置此属性。欲了解更多信息, 请参阅:ref:`api_guide_Name`。
+
+
+返回
+::::::::::
+
+    - ``out`` (Tensor): 形状为 ``[batch_size, seq_len, num_heads, head_dim]`` 的 4 维张量。数据类型可以是 float16 或 bfloat16。
+    - ``softmax`` (Tensor): 如果 return_softmax 为 False,则为 None。
```
Comment (Xreki) [inline docs/api/paddle/nn/functional/scaled_dot_product_attention_cn.rst:36]: 该API并没有`return_softmax`参数，也没有返回`softmax`。

Comment (luotao1) [reply docs/api/paddle/nn/functional/scaled_dot_product_attention_cn.rst:36]: 更新在
- https://github.com/PaddlePaddle/docs/pull/6484
